src/Frontend/css/App.css

```diff
@@ -1,5 +1,7 @@
+
 .App {
-  width: 100%;
-  height: 100%;
-  background: #f8f8f8;
+  display: flex;
+  background: #152536;
```

Comment:
Please revert to orginal.
.App {
width: 100%;
height: 100%;
background: #f8f8f8;
}